layers/+themes/theming/README.org

```diff
@@ -28,6 +28,30 @@ of the following form:
            (face2 attributes...)
            ...)
    ...)
+   
+   
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
```

Comment:
Doesn't belong here, probably got here by mistake. The new empty lines are unnecessary as well.